bundles/io/org.eclipse.smarthome.io.transport.serial/src/main/java/org/eclipse/smarthome/io/transport/serial/ProtocolType.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.io.transport.serial;
+
+import java.net.URI;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Holds the {@link PathType}, which specifies whether its a local or remote path and the scheme.
+ *
+ * @author Matthias Steigenberger - Initial Contribution
+ *
+ */
+@NonNullByDefault
+public class ProtocolType {
```

Comment:
The setters are not used at the moment (AFAIK). I don't know about a reason why changing a once constructed `ProtocolType` instance (modify pathType or scheme). Can we remove the setters and make the members final?
It would make the instances also thread safe (immutable objects).